app/public/css/styles.css

```diff
@@ -1149,6 +1129,42 @@ p {
   margin: 0;
 }
 
+/* INPUTS */
+
+.input {
+  border-radius: 4px;
+  border: 1px solid var(--color-border);
+  padding: 12px 16px;
+  font-size: 14px;
+  min-height: 58px;
```

Comment:
Why `min-height`? Should it have more than one line?

Reply:
Replaced with the `line-height` solution as discussed